pkg/server/router/keystore.go

```diff
@@ -103,7 +104,13 @@ type GetKeyDetailsResponse struct {
 	ID         string         `json:"id,omitempty"`
 	Type       crypto.KeyType `json:"type,omitempty"`
 	Controller string         `json:"controller,omitempty"`
-	CreatedAt  string         `json:"createdAt,omitempty"`
+
+	// Represents the time at which the key was created. Encoded according to RFC3339.
+	CreatedAt string `json:"createdAt,omitempty"`
+
+	// The public key in JWK format according to RFC7517. This public key is associated with the private
+	// key with the associated ID.
+	PublicKeyJWK jwx.PublicKeyJWK `json:"publicKeyJwk"`
```

Comment:
we should be careful here since not all key types support jwk. for example we don't have a way (yet) to represent BLS curves as JWKs 

this is probably fine for now since we don't have BBS+ in the service but this could be JWK or PublicKeyBase58